app/src/main/kotlin/com/wire/android/mapper/message/content/MessageContentMapper.kt

```diff
@@ -0,0 +1,53 @@
+package com.wire.android.mapper.message.content
+
+import androidx.annotation.StringRes
+import com.wire.android.R
+import com.wire.android.ui.home.conversations.findUser
+import com.wire.android.ui.home.conversations.model.UIMessageContent
+import com.wire.android.util.dispatchers.DispatcherProvider
+import com.wire.kalium.logic.data.message.Message
+import com.wire.kalium.logic.data.user.User
+import kotlinx.coroutines.withContext
+import javax.inject.Inject
+
+class MessageContentMapper @Inject constructor(
```

Comment:
generic content mapper that either maps regularMessageContent or a systemMessageContent